x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curations_logic.test.ts

```diff
@@ -0,0 +1,177 @@
+      },
+    },
+    results: [
+      {
+        id: 'some-curation-id',
+        last_updated: 'January 1, 1970 at 12:00PM',
+        queries: ['some query'],
+        promoted: [],
+        hidden: [],
+        organic: [],
+      },
+    ],
+  };
+
+  const DEFAULT_VALUES = {
+    dataLoading: true,
+    curations: [],
+    meta: DEFAULT_META,
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('has expected default values', () => {
+    mount();
+    expect(CurationsLogic.values).toEqual(DEFAULT_VALUES);
+  });
+
+  describe('actions', () => {
+    describe('onCurationsLoad', () => {
+      it('should set curations and meta state, & dataLoading to false', () => {
+        mount();
+
+        CurationsLogic.actions.onCurationsLoad(MOCK_CURATIONS_RESPONSE);
+
+        expect(CurationsLogic.values).toEqual({
+          ...DEFAULT_VALUES,
+          curations: MOCK_CURATIONS_RESPONSE.results,
+          meta: MOCK_CURATIONS_RESPONSE.meta,
+          dataLoading: false,
+        });
+      });
+    });
+
+    describe('onPaginate', () => {
+      it('should set meta.page.current state', () => {
+        mount();
+
+        CurationsLogic.actions.onPaginate(3);
+
+        expect(CurationsLogic.values).toEqual({
+          ...DEFAULT_VALUES,
+          meta: { page: { ...DEFAULT_VALUES.meta.page, current: 3 } },
+        });
+      });
+    });
+  });
+
+  describe('listeners', () => {
+    describe('loadCurations', () => {
+      it('should set dataLoading state', () => {
+        mount({ dataLoading: false });
+
+        CurationsLogic.actions.loadCurations();
+
+        expect(CurationsLogic.values).toEqual({
+          ...DEFAULT_VALUES,
+          dataLoading: true,
+        });
+      });
+
+      it('should make an API call and set curations & meta state', async () => {
+        http.get.mockReturnValueOnce(Promise.resolve(MOCK_CURATIONS_RESPONSE));
+        mount();
+        jest.spyOn(CurationsLogic.actions, 'onCurationsLoad');
+
+        CurationsLogic.actions.loadCurations();
+        await nextTick();
+
+        expect(http.get).toHaveBeenCalledWith('/api/app_search/engines/some-engine/curations', {
+          query: {
+            'page[current]': 1,
+            'page[size]': 10,
+          },
+        });
+        expect(CurationsLogic.actions.onCurationsLoad).toHaveBeenCalledWith(
+          MOCK_CURATIONS_RESPONSE
+        );
+      });
+
+      it('handles errors', async () => {
+        http.get.mockReturnValueOnce(Promise.reject('error'));
+        mount();
+
+        CurationsLogic.actions.loadCurations();
+        await nextTick();
+
+        expect(flashAPIErrors).toHaveBeenCalledWith('error');
+      });
+    });
+
+    describe('deleteCurationSet', () => {
+      const confirmSpy = jest.spyOn(window, 'confirm');
+
+      beforeEach(() => {
+        confirmSpy.mockImplementation(jest.fn(() => true));
+      });
+
+      it('should make an API call and set curations & meta state', async () => {
```

Comment:
Is this description accurate, does it actually set curations and meta state after deleting a curation?

Reply:
```suggestion
      it('should make an API call and show a success message', async () => {
```

Reply:
Awesome catch, thanks!

Reply:
https://github.com/elastic/kibana/pull/91565/commits/5d6fbe30f6b9e2bbffeb1f1eec7c4d56eb8f1aea